### PR TITLE
Updated UI of Onboarding screen

### LINKED ISSUE #23: Update UI of Onboarding page
image

### PR DESCRIPTION
**Pull request by ritvij14**
## Related Issues
Fixes #23 

## Tasks done:
- Added `TextEditingController` for updating username in `Store` and changing screens.
- Added SINIX Logo
- Added Enter button
- Added extra print statement which works as a check to know that the `TextField` and button are working fine.

## Current looks

https://appetize.io/app/89kqupyn9rnxmc43y59ejb9m5m?device=nexus5&scale=75&orientation=landscape&osVersion=8.1

## Check for the correct working of inputs:
On entering my username and pressing `Enter` this is shown as output:

image

### CODE REVIEW

**Comment by ritvij14** [comment]
I added an extra print statement just as a check, should I remove it?

**Comment by sanket143** [comment]
> I added an extra print statement just as a check, should I remove it?

No, it's fine. You can keep it if you think it'll be helpful.

File: lib/pages/onBoarding_page.dart [hunk 1]
```diff
@@ -21,30 +29,44 @@ class _OnBoardingPageState extends State<OnBoardingPage> {
             child: Column(
```
Comment (sanket143) [inline lib/pages/onBoarding_page.dart:29]: make its `MainAxisAlignment` to `center`

File: lib/pages/onBoarding_page.dart [hunk 2]
```diff
@@ -21,30 +29,44 @@ class _OnBoardingPageState extends State<OnBoardingPage> {
...
-                  size: 105,
+                  height: 70,
                 ),
+                Image.asset('assets/icon/logo_light.png'),
                 SizedBox(
-                  height: 60,
+                  height: 50,
                 ),
                 SizedBox(
                   width: 300,
```
Comment (sanket143) [inline lib/pages/onBoarding_page.dart]: can you make it `364`.

File: lib/pages/onBoarding_page.dart [hunk 3]
```diff
@@ -21,30 +29,44 @@ class _OnBoardingPageState extends State<OnBoardingPage> {
...
-                      Store.to.saveUserName(userName);
-                      Get.to(DiscoverDevices());
-                    },
+                    textAlign: TextAlign.center,
+                    controller: controller,
                     decoration: InputDecoration(
-                      labelText: "Enter Username",
+                      hintText: "USERNAME",
                       border: OutlineInputBorder(
                         borderRadius: BorderRadius.circular(16),
```
Comment (sanket143) [inline lib/pages/onBoarding_page.dart]: update this `borderRadius` to `3`

File: lib/pages/onBoarding_page.dart [hunk 4]
```diff
@@ -21,30 +29,44 @@ class _OnBoardingPageState extends State<OnBoardingPage> {
...
+                  height: 25.0,
+                ),
+                FlatButton(
+                  onPressed: () {
+                    Store.to.saveUserName(controller.text);
+                    print('${controller.text}');
+                    Get.to(DiscoverDevices());
+                  },
+                  shape: RoundedRectangleBorder(
+                    borderRadius: BorderRadius.circular(15.0)
```
Comment (sanket143) [inline lib/pages/onBoarding_page.dart]: update this `borderRadius` to `10`

File: lib/pages/onBoarding_page.dart [hunk 5]
```diff
@@ -21,30 +29,44 @@ class _OnBoardingPageState extends State<OnBoardingPage> {
...
+                  ),
+                  color: Colors.red[600],
```
Comment (sanket143) [inline lib/pages/onBoarding_page.dart]: Use `#DC143C` color.

**Review by sanket143 — CHANGES_REQUESTED** [review]
I'm sorry, I should've shared the Design constraints. For now, can you make these changes.

File: lib/pages/onBoarding_page.dart (hunk 5 above)
Comment (ritvij14) [reply lib/pages/onBoarding_page.dart]: Done.

File: lib/pages/onBoarding_page.dart (hunk 4 above)
Comment (ritvij14) [reply lib/pages/onBoarding_page.dart]: Done

File: lib/pages/onBoarding_page.dart (hunk 3 above)
Comment (ritvij14) [reply lib/pages/onBoarding_page.dart]: Done.

File: lib/pages/onBoarding_page.dart (hunk 2 above)
Comment (ritvij14) [reply lib/pages/onBoarding_page.dart]: Done :+1:

File: lib/pages/onBoarding_page.dart (hunk 1 above)
Comment (ritvij14) [reply lib/pages/onBoarding_page.dart:29]: Done :+1:

**Comment by ritvij14** [comment]
@sanket143 done with the changes that you asked, please have a look.

**Comment by sanket143** [comment]
give @ritvij14 75 points

**Pull request merged**